### PR TITLE
Center and Club rating

### PR DESCRIPTION
**Pull request by OH72**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/main/java/com/softserve/teachua/controller/ClubController.java [hunk 1]
```diff
@@ -30,12 +25,10 @@
     private static final int CLUBS_PER_USER_PAGE = 3;
 
     private final ClubService clubService;
-    private final JwtProvider jwtProvider;
```
Comment (OH72) [inline src/main/java/com/softserve/teachua/controller/ClubController.java:33]: Useless variable

File: src/main/java/com/softserve/teachua/config/SecurityConfig.java [hunk 2]
```diff
@@ -138,13 +138,15 @@ protected void configure(HttpSecurity http) throws Exception {
...
                 .antMatchers(HttpMethod.POST, "/api/club").hasAnyRole("MANAGER", "ADMIN")
+                .antMatchers(HttpMethod.PATCH, "/api/clubs/rating").hasRole("ADMIN")
                 .antMatchers(HttpMethod.POST, "/api/upload-excel").permitAll() // TODO: only for admins
                 .antMatchers(HttpMethod.POST, "/api/load-excel-to-db").permitAll() // TODO: only for admins
                 .antMatchers(HttpMethod.GET, "/api/download-database-sql").permitAll() // TODO: only for admins
                 .antMatchers(HttpMethod.PUT, "/api/club").hasAnyRole("MANAGER", "ADMIN")
                 .antMatchers(HttpMethod.POST, "/api/complaint", "/api/feedback").hasAnyRole("USER", "MANAGER", "ADMIN")
                 .antMatchers(HttpMethod.GET, "/api/center/**", "/api/centers/**", "/api/feedbacks/**").permitAll()
                 .antMatchers(HttpMethod.DELETE, "/api/center/**").hasAnyRole("MANAGER", "ADMIN")
+                .antMatchers(HttpMethod.PATCH, "/api/centers/rating").hasRole("ADMIN")
```
Comment (OH72) [inline src/main/java/com/softserve/teachua/config/SecurityConfig.java:149]: permitted admin to update rating for all centers.

File: src/main/java/com/softserve/teachua/config/SecurityConfig.java [hunk 3]
```diff
@@ -138,13 +138,15 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers(HttpMethod.GET, "/api/newslist", "/api/newslist/search").permitAll()
                 .antMatchers(HttpMethod.GET, "/api/club/**", "/api/clubs/**").permitAll()
                 .antMatchers(HttpMethod.POST, "/api/club").hasAnyRole("MANAGER", "ADMIN")
+                .antMatchers(HttpMethod.PATCH, "/api/clubs/rating").hasRole("ADMIN")
```
Comment (OH72) [inline src/main/java/com/softserve/teachua/config/SecurityConfig.java:141]: permitted admin to update rating for all clubs.

**Review by VictoriaAmazonka — APPROVED** [review]
If requirements are satisfied, I am ok to approve

**Pull request closed without merging**